File: JavaScript/Endpoint Examples/Multipart Payload/up-toolkit.js
// This request demonstrates how to check the status of the API servers
const axios = require("axios");

let config = {
  method: "get",
  maxBodyLength: Infinity, // set maximum length of the request body
  url: "https://api.pdfrest.com/up-toolkit", // up-forms and up-office can be used to query the other tools
  headers: {},
};

// define configuration options for axios request
axios
  .request(config)
  .then((response) => {
    console.log(JSON.stringify(response.data));
  })
  .catch((error) => {
    console.log(error);
  });

File: Python/Endpoint Examples/Multipart Payload/up-toolkit.py
import requests
import json

# up-forms and up-office can be used to query the other tools
up_url = f"https://api.pdfrest.com/up-toolkit"

print("Sending GET request to /up-toolkit endpoint...")
response = requests.get(up_url)

print("Response status code: " + str(response.status_code))

if response.ok:
    response_json = response.json()
    print(json.dumps(response_json, indent = 2))
else:
    print(response.text)
